exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -77,7 +77,7 @@ private ExecConstants() {
   public static final String SPILL_DIRS = "drill.exec.spill.directories";
 
   public static final String OUTPUT_BATCH_SIZE = "drill.exec.memory.operator.output_batch_size";
-  public static final LongValidator OUTPUT_BATCH_SIZE_VALIDATOR = new RangeLongValidator(OUTPUT_BATCH_SIZE, 1024, 512 * 1024 * 1024);
+  public static final LongValidator OUTPUT_BATCH_SIZE_VALIDATOR = new RangeLongValidator(OUTPUT_BATCH_SIZE, 1, 512 * 1024 * 1024);
```

Comment:
Maybe add a comment to explain the units here. Bytes? MB? A minimum batch size of 1 byte seems small, but a max size of 512 GB seems large, so not sure of the limits...

Reply:
Theoretically, 2Gb-1 should be the largest value as our value vectors use integers as offset values.